mfr/extensions/tabular/render.py

```diff
@@ -80,6 +80,12 @@ def _populate_data(self, fp, ext):
                         extension=self.metadata.ext,
                         formatting_function=str(function),
                     )
+                except ValueError:
```

Comment:
There are two types of exceptions here: `KeyError` and `ValueError`. I am able to trigger the latter by using the `error.dta`.

1. Is `ValueError` only applicable to `.dta` files? We should have a more general message saying the file can't be rendered. Instead of hard-coding `as .dta`, we can use a format string.
2. If both `KeyError` and `ValueError` are triggered by invalid file format, we can combine  them together.

Reply:
Going with #2 here, seems to make the most sense to me here. `error.dta` was a renamed csv. So it really is a formatting error.